Sprint5 maincontainer

CanoodlingSocks opened the pull request:
- Kopplat på 5 knappar på Navbaren varav Item-4 är en placeholder och det är fritt fram att ändra den

- All data (app) renderas inuti main-container
- Index.css har blivit uppdelad i olika stycken beroende på relevans. Här finns css för dom append:ade elementen som skapades upp i ListAllExpensesInBudget.js

- Slopade idéen för sidebar men utkommenterad kod för en sidebar button finns kvar i ListAllExpensesInBudget.js, var snäll och ha kvar den utkommenterade koden

- La till en simpel css animation för Välkomst texten i Home ;-)

Mvh Daniela
----
"Kachow!" - Blixten McQueen

CanoodlingSocks committed:
Add CSS , Navbar Btn Functionality

CanoodlingSocks committed:
Add sidebar.js

CanoodlingSocks committed:
some changes

CanoodlingSocks committed:
Css changes

CanoodlingSocks committed:
Add sidebar button to Budget

CanoodlingSocks committed:
Add some css to ListAllExpensesInBudget

EmilJohansson90 reviewed (changes requested):
Merge into dev instead of main.

Bergewing reviewed (commented):
Isn't this in direct merge conflict with the new navbar system in header and the accompanied code?

Krezeb committed:
Merge branch 'dev' into sprint5-maincontainer

Krezeb committed:
Update index.html

Added following to start of file:
<!DOCTYPE html>
<html lang="en">

Krezeb committed:
Update header.js

Added following to page start:

import { changePage } from './router.js'

Krezeb committed:
Update index.css

Altered CSS to try and prevent bugs. Probably didn't catch them all. Will have to adjust CSS rules afterwards.

Krezeb reviewed (approved):
Kanske blir fel i footer vid CSS overrides med Inline vs Stylesheets. Har justerat lite och löst lite conflicts. Vi testar med det. Kan lösa ev. buggar imorgon.

Saru444 reviewed (approved):
ser bra ut
